dotCMS/src/main/java/com/dotcms/rest/api/v1/system/ConfigurationResource.java

```diff
@@ -48,13 +59,92 @@
 
 	private final ConfigurationHelper helper;
 
+	private static final Pattern BLACK_LIST = Try.of(()->Pattern.compile(
+			Config.getStringProperty("OBFUSCATE_SYSTEM_ENVIRONMENTAL_VARIABLES", "passw|pass|passwd|secret|key|token"),
```

Comment:
I think we already have a `OBFUSCATE_SYSTEM_ENVIRONMENTAL_VARIABLES` defined in the JVMInfoResource. Wouldn't be convenient to have a public constant shared by both endpoints?